bibliopixel/drivers/SPI/WS2801.py

```diff
@@ -0,0 +1,19 @@
+from .. channel_order import ChannelOrder
+from . base import SPIBase
+from ... import gamma
+
+
+class WS2801(SPIBase):
+    """Main driver for WS2801 based LED strips on devices like the Raspberry Pi and BeagleBone"""
+
+    def __init__(self, num, gamma=gamma.WS2801, **kwargs):
+        print("Loading WS2801")
+        spi_speed = kwargs.get('spi_speed', 1)
+        if spi_speed > 1 or spi_speed <= 0:
```

Comment:
Whaaaat? I did not know you could do that. Learn something knew every day I guess... DONE.